app/views/roles/_header.html.erb

```diff
@@ -1,4 +1,4 @@
-<header class="govuk-grid-row">
+<header class="govuk-grid-row" dir="<%= page_text_direction %>" lang="<%= role.locale %>">
```

Comment:
This shouldn't be rtl since it contains a left to right language - in this case, 'Ministerial role'.